types/lnrpc.d.ts

```diff
@@ -426,18 +524,27 @@ export interface SendRequest {
   paymentRequest?: string;
   finalCltvDelta?: number;
   feeLimit?: FeeLimit;
+  outgoingChanId?: string;
+  cltvLimit?: number;
 }
 
 export interface SendResponse {
   paymentError: string;
   paymentPreimage: Buffer | string;
   paymentRoute?: Route;
+  paymentHash: Uint8Array | string;
```

Comment:
Should be `Buffer | string`. The generated types don't identify these correctly.